### ISSUE TITLE
"changes" view

### ISSUE THREAD

**Issue by clkao** [karissa/dat-explorer#33]
show entries from the metadata feed, and allow viewing an older version of an overwritten file.

**Comment by okdistribute** [comment]
yes! @mafintosh said that one day we could build a `archive.createHistoryStream()` function that would do most of the heavily lifting

**Comment by clkao** [comment]
depends on https://github.com/mafintosh/hyperdrive/issues/63

**Comment by okdistribute** [comment]
@mafintosh said that archive.list() does 95% of what we want, so we can probably go ahead and try to implement it with archive.list and see how it goes

**Comment by mafintosh** [comment]
:+1: its just missing the change number basically.
